syft/formats/common/spdxhelpers/to_format_model.go

```diff
@@ -516,10 +517,18 @@ func toFileTypes(metadata *source.FileMetadata) (ty []string) {
 	return ty
 }
 
+// other licenses are for licenses from the pkg.Package that do not have an SPDXExpression
+// field. The spdxexpression field is only filled given a validated Value field.
 func toOtherLicenses(catalog *pkg.Collection) []*spdx.OtherLicense {
 	licenses := map[string]bool{}
 	for _, p := range catalog.Sorted() {
-		for _, license := range parseLicenses(p.Licenses) {
+		d, c := parseLicenses(p.Licenses)
```

Comment:
nit / opinion: `d` and `c` aren't very obvious names

Reply:
updated to be explicit with `declaredLicenses` and `concludedLicenses`